index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Raj Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <main class="rj">
+    <header>
+      <div>      <a href="#" class="logo">My Logo</a></div>
+
+      <div>
+        <img src="image/Union.png" alt="button" />
+      
+    </div>
+
+    </header>
+    <section>
+      <h1>I'm Vivek Raj glad to see you !</h1>
+      <p class="software">
+        I’m a software developer! I can help you build a product, 
+        feature or website Look through some of my work and 
+        experience! If you like what you see 
+        and have a project you need coded, 
+        don’t hestiate to contact me.
+      </p>
+      <p class="connect">LET'S CONNECT</p>
+      <div class="media">
```

Comment:
- [ ] I think it is better to use` hx` tag like `h3,h4,h5` tag for Let's connect as this is heading for social media icons.